source/engine/graphics/api/vulkan/RenderGraph.cpp

```diff
@@ -164,13 +167,18 @@ void RenderGraph::SinkPostProcessImages()
     OPTICK_CATEGORY("RenderGraph::SinkPostProcessImages", Optick::Category::Rendering);
 
     auto* currentFrame = m_frameManager->CurrentFrameContext();
-    auto& perFrameShadowMapView = m_postProcessImageViews.at(PostProcessImageType::ShadowMap).perFrameViews[currentFrame->Index()];
-    perFrameShadowMapView = std::vector<vk::ImageView>(m_maxLights, m_dummyShadowMap.view.get());
+    auto& perFrameShadowMapViews = m_postProcessImageViews.at(PostProcessImageType::ShadowMap).perFrameViews[currentFrame->Index()];
+    perFrameShadowMapViews = std::vector<vk::ImageView>(m_maxLights, m_dummyShadowMap.view.get());
     auto& shadowPasses = m_shadowMappingPasses[currentFrame->Index()];
 
-    for (auto i : std::views::iota(0u, m_activeShadowMappingPasses))
+    for (auto i : std::views::iota(0u, m_omniDirLightCount))
     {
-        perFrameShadowMapView[i] = shadowPasses[i].GetAttachmentView(0u);
+        perFrameShadowMapViews[i] = shadowPasses[i].GetAttachmentView(0u);
+    }
+
+    for (auto i : std::views::iota(m_omniDirLightCount, m_omniDirLightCount + m_uniDirLightCount))
```

Comment:
Can these two for loops be combined? To my eyes, its one sequential pass split into two steps.